build.gradle

```diff
@@ -10,20 +10,12 @@ buildscript {
     }
     dependencies {
         classpath 'com.vanniktech:gradle-maven-publish-plugin:0.9.0-SNAPSHOT'
-        classpath 'com.android.tools.build:gradle:3.5.0'
+        classpath 'com.android.tools.build:gradle:3.5.1'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
         classpath "org.jetbrains.dokka:dokka-android-gradle-plugin:$dokka_version"
     }
 }
 
-plugins {
```

Comment:
Just because I haven't used it in months, and it just adds clutter. It's not even running on CircleCI. I will investigate what the best options are whenever we add more static analysis.